modules/machine-config-node-disruption-example.adoc

```diff
@@ -80,7 +96,7 @@ spec:
 # ...
 ----
 
-In the following example, when changes are made to the `/etc/chrony.conf` file, the MCO reloads the `chronyd.service` on the cluster nodes.
+In the following example, when changes are made to the `/etc/chrony.conf` file, the MCO reloads the `chronyd.service` on the cluster nodes. If files are added to or modified in the `/var/run` directory, the MCO applies the changes with no further action.
```

Comment:
Nit: I think it might be worth mentioning that if two policies applies to a certain file, the closest path match would be used. This is now possible because we support directories. 

For example in this case, if there is another policy for `/var/run/mco` and if the change is in the `/var/run/mco/` directory, the` /var/run/mco` policy would apply over the `/var/run/ ` policy.

Of course, this could be incredibly obvious, in which case feel to ignore me (:

Reply:
@djoshy What do you  mean by "the closest path match"? It is the "more similar" path?

Reply:
Yep! Just so we are on the same page I want to clarify the policy you defined for `/var/run` is not just for files in that directory. It is for everything you create in that directory, including subdirectories that may exist in `/var/run`.

Now, adding on to my example, a file located at `/var/run/mco/test_file` will have both policies apply to it as it i technically in both directories at the same time. In that case, the closest path match is the more specific one of the two policies.

Reply:
@djoshy  Do you see this as a common use case? 

> a file located at /var/run/mco/test_file will have both policies apply to it as it in technically in both directories at the same time

Reply:
It's possible, but I don't have much insight into customer use cases as the feature just GAed. I can see someone wanting a general policy for a certain directory and slightly different one for a very specific file in that directory. It's really more of a nit, and may be pretty obvious to a user using this level of customization. My vote is that it is worth mentioning, but not a big deal if we don't.

Reply:
How does [this sound](https://github.com/openshift/openshift-docs/pull/81321/commits/896e63ed5c396010cb9191c71b17195786be6bfe#diff-77b1ee889ec4b3d828092d6287889f04e705b32c4b3d3db7f483396eee25d6beR30-R37)?

Reply:
Looks good, just added a clarification that only the closest match will be executed